android/src/main/java/leesiongchan/reactnativeescpos/PrinterService.java

```diff
@@ -3,25 +3,26 @@
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 import android.net.Uri;
-
 import com.google.zxing.BarcodeFormat;
 import com.google.zxing.WriterException;
 import com.google.zxing.common.BitMatrix;
 import com.google.zxing.qrcode.QRCodeWriter;
-
+import io.github.escposjava.print.Commands.*;
```

Comment:
Do we need this? We have already import it in line 25

Reply:
remove it.. I don know how it appear here